ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -1374,12 +1389,18 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
               <div>You can also make changes to your answers after you create your workspace.</div>
             </ModalBody>
             <ModalFooter>
-              <Button type='secondary' style={{marginRight: '1rem'}}
```

Comment:
Here begins the actual fix.  "disabled if errors or loading"

Reply:
Nice. It would be handy to have a reusable pattern for this kind of control.